sdk/dns/Azure.ResourceManager.Dns/src/Customization/RestOperation/NaptrRecordRestOperations.cs

```diff
@@ -0,0 +1,779 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.Dns.Models;
+
+namespace Azure.ResourceManager.Dns
+{
+    internal partial class DnsNaptrRecordRestOperations
+    {
+        private readonly TelemetryDetails _userAgent;
+        private readonly HttpPipeline _pipeline;
+        private readonly Uri _endpoint;
+        private readonly string _apiVersion;
+
+        /// <summary> Initializes a new instance of RecordSetsRestOperations. </summary>
+        /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
+        /// <param name="applicationId"> The application id to use for user agent. </param>
+        /// <param name="endpoint"> server parameter. </param>
+        /// <param name="apiVersion"> Api Version. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="pipeline"/> or <paramref name="apiVersion"/> is null. </exception>
+        public DnsNaptrRecordRestOperations(HttpPipeline pipeline, string applicationId, Uri endpoint = null, string apiVersion = default)
+        {
+            _pipeline = pipeline ?? throw new ArgumentNullException(nameof(pipeline));
+            _endpoint = endpoint ?? new Uri("https://management.azure.com");
+            _apiVersion = apiVersion ?? "2018-05-01";
```

Comment:
This should be "2023-07-01-preview"

Reply:
Hi @rsh-msft , I raised a PR to update these files: https://github.com/Azure/azure-sdk-for-net/pull/37742, could  please you take a look, thanks.

Reply:
Hi @rsh-msft I raised a PR to update these files: https://github.com/Azure/azure-sdk-for-net/pull/37745, could please you take a look, thanks.

Reply:
@dvbb  LGTM. thanks for making the changes